CHANGELOG.rst

```diff
@@ -35,6 +35,8 @@ Improved Documentation
   xdist could identify the wrong test as failing when test crashes at teardown.
   To address this issue, xdist now marks test as complete at teardown.
 
+- 168.trivial: add long option `--numprocesses` as alternative for `-n`.
```

Comment:
Yep, the `CHANGELOG.rst` file should not be modified (there's a comment about it at the top of the file), it is generated from the fragments in the   `changelog` folder upon release. This eliminates changelog conflicts. 😁 

I updated the [PR template](https://github.com/pytest-dev/pytest-xdist/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to make that clearer, feel free to comment on it.